config/docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:8
+
+RUN apt-get update && apt-get install -y postgresql-client
+
+RUN npm install -g lerna yarm flow-bin
```

Comment:
Should that be yarn?

Reply:
ahh in fact it shouldn’t be there at all, you just made me realize I’ve installed another package by mistake ^^